tests/src/test/java/alluxio/client/cli/fsadmin/command/ReportCommandIntegrationTest.java

```diff
@@ -53,8 +53,20 @@ public void reportConfiguration() {
     String output = mOutput.toString();
     Assert.assertThat(output,
         CoreMatchers.containsString("Alluxio configuration information:"));
-    Assert.assertThat(output,
-        CoreMatchers.containsString("alluxio.test.mode"));
+
+    // Output should not contain raw values with ${VALUE} format
+    Assert.assertFalse(output.contains("$")
```

Comment:
better to use regexp to decide if we have `${VALUE}` inside.
https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html